packages/gcf-utils/src/gcf-utils.ts

```diff
@@ -57,14 +57,14 @@ interface Scheduled {
 
 interface EnqueueTaskParams {
   body: string;
-  signature: string;
   id: string;
   name: string;
 }
 
 export interface WrapOptions {
   background: boolean;
   logging: boolean;
+  skipVerification?: boolean;
```

Comment:
Please add a comment for `skipVerification`.  Comments for `background` and `logging` would also be appreciated.

Reply:
Added comments for each